README.md

```diff
@@ -32,143 +38,191 @@ Rouge::Themes::Base16.mode(:light).render(scope: '.highlight')
 
-* `Rouge::Formatters::HTML.new` - will render your code with standard class names for tokens,
-  with no div-wrapping or other bells or whistles.
-* `Rouge::Formatters::HTMLInline.new(theme)` - will render your code with no class names, but
-  instead inline the styling options into the `style=` attribute. This is good for emails and
-  other systems where CSS support is minimal.
-* `Rouge::Formatters::HTMLLinewise.new(formatter, class: 'line-%i')`
-  This formatter will split your code into lines, each contained in its own div. The
-  `class` option will be used to add a class name to the div, given the line
-  number.
-* `Rouge::Formatters::HTMLLineTable.new(formatter, opts={})` will output an HTML table containing
-  numbered lines, each contained in its own table-row. Options are:
+* `Rouge::Formatters::HTMLLinewise.new(formatter, class: 'line-%i')` will split
+  your code into lines, each contained in its own div. The `class` option will
+  be used to add a class name to the div, given the line number.
+
+* `Rouge::Formatters::HTMLLineTable.new(formatter, opts={})` will output an HTML
+  table containing numbered lines, each contained in its own table-row. Options
+  are:
     * `start_line: 1` - the number of the first row
-    * `line_id: 'line-%i'` - a `sprintf` template for `id` attribute with current line number
+    * `line_id: 'line-%i'` - a `sprintf` template for `id` attribute with
+      current line number
     * `line_class: 'lineno'` - a CSS class for each table-row
     * `table_class: 'rouge-line-table'` - a CSS class for the table
     * `gutter_class: 'rouge-gutter'` - a CSS class for the line-number cell
     * `code_class: 'rouge-code'` - a CSS class for the code cell
-* `Rouge::Formatters::HTMLPygments.new(formatter, css_class='codehilite')`
-  wraps the given formatter with div wrappers generally expected by stylesheets designed for
-  Pygments.
-* `Rouge::Formatters::HTMLTable.new(formatter, opts={})` will output an HTML table containing
-  numbered lines. Options are:
+
+* `Rouge::Formatters::HTMLPygments.new(formatter, css_class='codehilite')` wraps
+  the given formatter with div wrappers generally expected by stylesheets
+  designed for Pygments.
+
+* `Rouge::Formatters::HTMLTable.new(formatter, opts={})` will output an HTML
+  table containing numbered lines. Options are:
     * `start_line: 1` - the number of the first line
     * `line_format: '%i'` - a `sprintf` template for the line number itself
     * `table_class: 'rouge-table'` - a CSS class for the table
     * `gutter_class: 'rouge-gutter'` - a CSS class for the gutter
     * `code_class: 'rouge-code'` - a CSS class for the code column
-* `Rouge::Formatters::HTMLLegacy.new(opts={})` is a backwards-compatibility class intended
-  for users of rouge 1.x, with options that were supported then. Options are:
+
+* `Rouge::Formatters::HTMLLegacy.new(opts={})` is a backwards-compatibile class
```

Comment:
noun instead of adjective.
```suggestion
* `Rouge::Formatters::HTMLLegacy.new(opts={})` is a backwards-compatibility class
```

Reply:
Is this wrong? The adjective is modifying 'class'. I'm not sure why the compound noun 'backwards-compatibility class' is to be preferred. (I did misspell 'compatible' which is a little embarrassing.)

Reply:
The prime function of `Rouge::Formatters::HTMLLegacy` is to *maintain backwards-compatibility*. Saying that it is *backwards-compatible* doesn't really imply that the class exists for the sole reason to provide *backwards-compatibility* with an older API.

Reply:
Makes sense 👍